.github/workflows/deploy-on-pr-merge.yaml

```diff
@@ -26,17 +33,21 @@ jobs:
         with:
           java-version: 11
           distribution: 'adopt'
-          server-id: matsim-snapshots
+          server-id: ${{ github.event_name == 'push' && 'matsim-snapshots' || 'matsim-releases' }} #choose mvn repo
           server-username: MAVEN_USERNAME
           server-password: MAVEN_PASSWORD
 
+      - name: Set MATSim version (if PR-labelled version)
+        if: github.event_name == 'pull_request'
+        run: mvn versions:set --batch-mode -DnewVersion=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout | sed 's/SNAPSHOT//')PR${{ github.event.pull_request.number }} -DgenerateBackupPoms=false
+
       # Build and publish are separated so we start deploying only after all jars are built successfully
       - name: Build jars
-        run: mvn -pl -distribution package --batch-mode -Dmaven.javadoc.skip -Dassembly.skipAssembly=true -DskipTests
+        run: mvn package --batch-mode -DskipTests
```

Comment:
This line should not be replaced with the one from `master` as the mvn project structure has changed in the meantime.